x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_snooze/panel/base_snooze_panel.tsx

```diff
@@ -318,27 +329,37 @@ export const BaseSnoozePanel: React.FunctionComponent<BaseSnoozePanelProps> = ({
             </EuiFlexItem>
           </EuiFlexGroup>
           <EuiFlexGroup direction="column" gutterSize="xs" data-test-subj="ruleSchedulesList">
-            {scheduledSnoozes!.map((schedule) => (
-              <EuiFlexItem key={`snooze-${schedule.id}`}>
-                <button
-                  style={{ paddingLeft: '9px', paddingRight: '9px' }}
-                  className="euiButton euiPanel euiPanel--borderRadiusMedium euiPanel--subdued euiPanel--noShadow euiPanel--noBorder"
-                  onClick={onClickEditScheduleFactory(schedule as SnoozeSchedule)}
-                >
-                  <EuiFlexGroup alignItems="center">
-                    <EuiFlexItem grow={false}>
-                      <EuiIcon type="calendar" />
-                    </EuiFlexItem>
-                    <EuiFlexItem style={{ textAlign: 'left' }}>
-                      {scheduleSummary(schedule as SnoozeSchedule)}
-                    </EuiFlexItem>
-                    <EuiFlexItem grow={false}>
-                      <EuiIcon type="arrowRight" />
-                    </EuiFlexItem>
-                  </EuiFlexGroup>
-                </button>
-              </EuiFlexItem>
-            ))}
+            {scheduledSnoozes!
+              .filter((s) => s.id)
+              .map((schedule) => {
+                const isActive = schedule.id && activeSnoozes.includes(schedule.id);
+                return (
+                  <EuiFlexItem key={`snooze-${schedule.id}`}>
+                    <button
+                      style={{
+                        paddingLeft: '9px',
+                        paddingRight: '9px',
+                        color: isActive ? '#a8376a' : '#646a77',
+                        backgroundColor: isActive ? 'rgba(240,78,152,0.2)' : 'rgb(247, 248, 252)',
```

Comment:
nitpick: I think it would be nice if we used the `euiTheme` constants if they're available

Reply:
This is a workaround for styling these buttons the same as an `EuiPanel` with `color="accent"` or `="subdued"`. I'm able to get the colors for `subdued` from `euiTheme` but unfortunately `accent` is using some non-standard colors.

Passing in the classNames directly (`euiPanel--accent` or `--subuded`) doesn't actually work due to what appears to be some internal styled-components issue going on with EUI.